ion/bitstream.go

```diff
@@ -427,12 +430,18 @@ func (b *bitstream) ReadInt() (interface{}, error) {
 	default:
 		// Need to go big.Int.
 		i := new(big.Int).SetBytes(bs)
+		isZero = i.Cmp(big.NewInt(0)) == 0
```

Comment:
Constructing a bigint can be expensive as it often requires a heap allocation; is there a constant for `0` that we could re-use instead? If not, can we define one in this package?

Reply:
I couldn't find any constant value for 0, that's why I constructed bigInt 0.

Seems like big.Int is an Invalid constant type.
![bigInt](https://user-images.githubusercontent.com/57376624/85470509-fd0b7e00-b563-11ea-91a8-55c3c51a07e3.JPG)

I used [```BitLen()```](https://golang.org/pkg/math/big/#Int.BitLen) to check for zero ([Example in play.golang](https://play.golang.org/p/RAa7nzSgHyj))

Reply:
> I used BitLen() to check for zero

Clever workaround, I like it. :+1: